frontend/selectors/index.js

```diff
@@ -0,0 +1,19 @@
+import { createSelector } from 'reselect';
+import { getCurrentRoute } from '@shopgate/pwa-common/selectors/router';
+import getConfig from '../helpers/getConfig';
+
+const {
+  affectedPages,
+} = getConfig();
+
+// Change the header on every page as default
+let shouldChangeHeader = () => true;
+
+if (affectedPages && affectedPages.length) {
+  shouldChangeHeader = createSelector(
+    getCurrentRoute,
+    route => affectedPages.some(page => route.pattern.startsWith(page))
```

Comment:
If you configure "/" it will be true for ALL pages. Comparing pattern should be exact page === route.pattern.
SI should configure patterns in the extension config.

Reply:
done